sdk/keyvault/azure-keyvault-certificates/src/main/java/com/azure/security/keyvault/certificates/CertificateClient.java

```diff
@@ -0,0 +1,1259 @@
+     *
+     *<p>The {@link LifetimeAction} of type {@link LifetimeActionType#EMAIL_CONTACTS} set on a {@link CertificatePolicy} emails the contacts set on the vault when triggered.</p>
+     *
+     * <p><strong>Code Samples</strong></p>
+     * <p>Sets the certificate contacts in the Azure Key Vault. Prints out the returned contacts details.</p>
+     *
+     * {@codesnippet com.azure.security.keyvault.certificates.certificateClient.setCertificateContacts#contacts}
+     *
+     * @param contacts The list of contacts to set on the vault.
+     * @throws HttpRequestException when a contact information provided is invalid/incomplete.
+     * @return A {@link PagedIterable} containing all of the {@link Contact certificate contacts} in the vault.
+     */
+    @ServiceMethod(returns = ReturnType.COLLECTION)
+    public PagedIterable<Contact> setCertificateContacts(List<Contact> contacts) {
+        return setCertificateContacts(contacts, Context.NONE);
+    }
+
+    /**
+     * Sets the certificate contacts on the key vault. This operation requires the {@code certificates/managecontacts} permission.
+     *
+     *<p>The {@link LifetimeAction} of type {@link LifetimeActionType#EMAIL_CONTACTS} set on a {@link CertificatePolicy} emails the contacts set on the vault when triggered.</p>
+     *
+     * <p><strong>Code Samples</strong></p>
+     * <p>Sets the certificate contacts in the Azure Key Vault. Prints out the returned contacts details.</p>
+     *
+     * {@codesnippet com.azure.security.keyvault.certificates.certificateClient.setCertificateContacts#contacts-context}
+     *
+     * @param contacts The list of contacts to set on the vault.
+     * @param context Additional context that is passed through the Http pipeline during the service call.
+     * @throws HttpRequestException when a contact information provided is invalid/incomplete.
+     * @return A {@link PagedIterable} containing all of the {@link Contact certificate contacts} in the vault.
+     */
+    @ServiceMethod(returns = ReturnType.COLLECTION)
+    public PagedIterable<Contact> setCertificateContacts(List<Contact> contacts, Context context) {
+        return new PagedIterable<>(client.setCertificateContacts(contacts, context));
+    }
+
+    /**
+     * Lists the certificate contacts in the key vault. This operation requires the certificates/managecontacts permission.
+     *
+     * <p><strong>Code Samples</strong></p>
+     * <p>Lists the certificate contacts in the Azure Key Vault. Prints out the returned contacts details in the response.</p>
+     *
+     * {@codesnippet com.azure.security.keyvault.certificates.certificateClient.listCertificateContacts}
+     *
+     * @return A {@link PagedIterable} containing all of the {@link Contact certificate contacts} in the vault.
+     */
+    @ServiceMethod(returns = ReturnType.COLLECTION)
+    public PagedIterable<Contact>
```

Comment:
Fix javadoc. This method doesn't return `Mono`.

Reply:
fixed.